File: Python/Others/Contacts Manager/Libraries_and_Variables_Definition/libraries_and_variables.py
import os
import pygame
import pickle
import pyautogui

pygame.init()

first_names = {}
middle_names = {}
last_names = {}
emails = {}
phone_numbers = {}
contact_database = []
path = 'contact_database/'

# white color
white = (255, 255, 255)
white_ = (100, 150, 200)

background_colour = (20, 100, 200)

fn_disp_coord = [0.28, 0.4, 0.4, 0.1, 0.1, 0.005]
mn_disp_coord = [0.28, 0.4, 0.4, 0.1, 0.08, 0.005]
ln_disp_coord = [0.28, 0.4, 0.4, 0.1, 0.1, 0.005]
e_disp_coord = [0.28, 0.4, 0.4, 0.1, 0.13, 0.005]
p_disp_coord = [0.28, 0.4, 0.4, 0.1, 0.13, 0.005]
id_disp_coord = [0.28, 0.4, 0.4, 0.1, 0.18, 0.005]

text1 = "Name contains Aphabates only.\n"
text2 = "Email must start with Alphanumeric key.\n"
text3 = "Phone number must be 10 digits.\n"
contact_invalid_text = text1 + text2 + text3


class Contact:
    id_, first_name, middle_name, last_name, email, phone = 0, "", "", "", "", ""

File: C and C++/Contacts Manager/libraries_and_variables.cpp
#include <map>
#include <vector>
#include <string>
#include <fstream>
#include <iostream>
#include <bits/stdc++.h>

using namespace std;

int ids;
unordered_map<string, int> phone_number;
vector<struct Contact> Contact_Database;
vector<vector<string>> Contact_Database_;
unordered_map<string, vector<int>> emails;
unordered_map<string, vector<int>> first_names;
unordered_map<string, vector<int>> middle_names;
unordered_map<string, vector<int>> last_names;
